### ISSUE TITLE
Accessibility Testing for Pension Burial Benefits, 21-0969, New form

### ISSUE THREAD

**Issue by eileen-coforma** [department-of-veterans-affairs/va.gov-team#113696]
### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@eileen-coforma

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")
- [ ] (Mobile app only) Color contrast ratios are met for all supported dark, contrast, and inverted color modes
- [ ] (Mobile app only) All elements retain their meaning for all supported dark, contrast, and inverted color modes

### How did color testing go?

No issues. Did not test mobile app, since it's a web-based form.

### Who completed the axe scans?

@eileen-coforma

### axe checks

- [x] Each page or screen has been scanned using axe (results shared below)
- [x] axe is integrated into your end-to-end testing

### axe DevTools scan results

Page title | Issue
-- | --
Review and submit | Missing H2s

### How did axe testing go?

Missing H1s and H2s, bug tickets have been written, and they're getting resolved.

### Who completed the content zoom and reflow test?

@eileen-coforma

### Content zoom and reflow checks

- [x] (Web only) All page elements are readable and usable at 200% zoom
- [x] (Web only) All page elements are readable and usable at 300% zoom
- [ ] (Web only) All page elements are readable and usable at 400% zoom
- [ ] (Mobile app only) All page elements are readable and usable when device text is at the maximum possible size

### How did content zoom and reflow testing go?

Overflow issues in the following pages:

- Royalties and other properties - Type - Text overflow
- Upload documents - File upload component delete modal overflow
- Review and submit - Privacy agreement link overflow

### Who completed the keyboard navigation test?

@eileen-coforma

### Keyboard navigation checks

- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [ ] All elements under focus have a visible focus indicator
- [x] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

No issues with keyboard testing
Missing focus on the following pages:

- [Veteran information](https://staging.va.gov/income-and-asset-statement-form-21p-0969/personal/information) (authenticated)

### Do you have any other results to share?

Have not done advanced accessibility testing.

**Comment by eileen-coforma** [comment]
Updated ticket based on certain bugs being resolved.
